pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -783,4 +809,23 @@ mod minting {
 				assert_ok!(AwesomeAvatars::mint(Origin::signed(ALICE), MintCount::One));
 			});
 	}
+
+	#[test]
+	fn mint_should_return_error_on_insuficient_founds() {
+		let season = Season::default().end(20);
+
+		ExtBuilder::default()
+			.organizer(ALICE)
```

Comment:
Let's be a bit more explicit here:
```suggestion
			.organizer(ALICE)
			.balances(vec![(ALICE, 1)])
			.mint_fees(MintFees { one: 11, three: 22, six: 33 })
```

Reply:
Added